### PR TITLE
Follow up on PR 2868 and `go mod tidy`

### PR DESCRIPTION
**Pull request by yongtang**
### 1. Why is this pull request needed and what does it do?
This fix is a follow up on comment https://github.com/coredns/coredns/pull/2868#discussion_r291609761 to inline short function `t.setupDatadog()`

This fix also runs `go mod tidy` to remove duplicate entries in go.mod.

### 2. Which issues (if any) are related?

PR #2868 

### 3. Which documentation changes (if any) need to be made?
n/a
### 4. Does this introduce a backward incompatible change or deprecation?
n/a

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,7 +4,6 @@ go 1.12
 
 require (
 	cloud.google.com/go v0.39.0 // indirect
-	github.com/DataDog/dd-trace-go v1.14.0
```
Comment (miekg) [inline go.mod:7]: `go.mod` changes should not be needed here?

**Review by miekg — CHANGES_REQUESTED** [review]
lgtm, apart from the comment I made

File: go.mod (hunk 1 above)
Comment (yongtang) [reply go.mod:7]: @miekg Ah it actually points to `gopkg.in/DataDog/dd-trace-go.v1 v1.14.0` now. See  Line 70 below.

Comment (miekg) [reply go.mod:7]: but do we want all the other updates here as well?

Comment (yongtang) [reply go.mod:7]: I think in PR #2868, three copies were added (v0.6.1 was the old one):

```
        github.com/DataDog/dd-trace-go v1.14.0
	gopkg.in/DataDog/dd-trace-go.v0 v0.6.1		
 	gopkg.in/DataDog/dd-trace-go.v1 v1.14.0
```

After `go mod tidy` in this PR, two are removed:
```diff
-       github.com/DataDog/dd-trace-go v1.14.0
-	gopkg.in/DataDog/dd-trace-go.v0 v0.6.1		
 	gopkg.in/DataDog/dd-trace-go.v1 v1.14.0
```

Comment (yongtang) [reply go.mod:7]: @miekg Do you mean you want to update `go mod` to bring everything update to date (minus client-go as it has issues with compatibility)?

**Comment by miekg** [comment]
No I'm just wondering if all the go.mod changes should be here at all if
you're just inlining that function. If you say they should then I'm happy
to merge

**Comment by yongtang** [comment]
@miekg I think `go mod tidy` change (removing two duplicate copies) should stay. Helps users to download 2 extra copies of DataDog dependencies (might be small though) in `go cache`.

**Comment by yongtang** [comment]
> I think go mod tidy change (removing two duplicate copies) should stay. Helps users to download 2 extra copies of DataDog dependencies (might be small though) in go cache.

@miekg sorry, is late night in my place so might be typing some errors in my last comment. I think the changes in go mod are good in this PR. It `avoids` users to download extra 2 copies of DataDog dependencies (typed `helps` not `avoid` in last comment).

**Comment by miekg** [comment]
I'll merge. Thanks!
Go to sleep :)

**Pull request merged**